scripts/validate_container.sh

```diff
@@ -170,49 +170,53 @@ do
-        # A simplified test for these use cases is included in the 'test/Remote-Sim/' test suite. 
-        # Skipped tests that require passing kernel callables to entry-point kernels for the "remote-mqpu" target.
-        elif [[ "$t" == "remote-mqpu" ]] && [[ "$ex" == *"vqe_h2"* || "$ex" == *"qaoa_maxcut"* || "$ex" == *"gradients"* || "$ex" == *"grover"* || "$ex" == *"multi_controlled_operations"* || "$ex" == *"phase_estimation"* ]];
-        then
-            let "skipped+=1"
-            echo "Skipping $ex for $t target.";
-            echo ":white_flag: Not executed for performance reasons. Test skipped." >> "$tmpFile_$(echo $t | tr - _)"
+            echo ":white_flag: $filename: Issue https://github.com/NVIDIA/cuda-quantum/issues/884. Test skipped." >> "${tmpFile}_$(echo $t | tr - _)"
 
-        elif [[ "$t" == "remote-mqpu" && "$mpi_available" == true && "$ssh_available" == false ]];
-        then
-            # Don't run remote-mqpu if the MPI installation is incomplete (e.g., missing an ssh-client).
-            let "skipped+=1"
-            echo "Skipping $t target due to incomplete MPI installation.";
-            echo ":white_flag: Incomplete MPI installation. Test skipped." >> "$tmpFile_$(echo $t | tr - _)"
+        elif [ "$t" == "remote-mqpu" ]; then
+            # Skipped long-running tests (variational optimization loops) for the "remote-mqpu" target to keep CI runtime managable.
+            # A simplified test for these use cases is included in the 'test/Remote-Sim/' test suite. 
+            # Skipped tests that require passing kernel callables to entry-point kernels for the "remote-mqpu" target.
+            if [[ "$ex" == *"vqe_h2"* || "$ex" == *"qaoa_maxcut"* || "$ex" == *"gradients"* || "$ex" == *"grover"* || "$ex" == *"multi_controlled_operations"* || "$ex" == *"phase_estimation"* ]];
+            then
+                let "skipped+=1"
+                echo "Skipping $t target.";
+                echo ":white_flag: $filename: Not executed for performance reasons. Test skipped." >> "${tmpFile}_$(echo $t | tr - _)"
+            # Don't run remote-mqpu if the MPI installation is incomplete (e.g., missing an ssh-client).            
+            elif [[ "$mpi_available" == true && "$ssh_available" == false ]];
+            then
+                let "skipped+=1"
+                echo "Skipping $t target due to incomplete MPI installation.";
+                echo ":white_flag: $filename: Incomplete MPI installation. Test skipped." >> "${tmpFile}_$(echo $t | tr - _)"
+            else
+                # Test with MLIR compilation enabled by default.
+                target_flag+=" --enable-mlir"
+            fi
 
         else
             echo "Testing on $t target..."
-            if [ "$t" == "default" ]; then 
-                nvq++ $ex && status=$?
-            else
-                nvq++ $ex --target $t && status=$?
-            fi
+            nvq++ $ex $target_flag && status=$?
```

Comment:
Won't `status=$?` get skipped if `nvq++` fails? Should it be a `;` instead of `&&`?